gpii/node_modules/flowManager/configs/gpii.flowManager.config.base.json

```diff
@@ -39,6 +40,10 @@
                     "dbName": "auth"
                 },
                 "target": "{that flowManager oauth2DataStore}.options.dataSourceConfig"
+            },
+            "payloadSizeLimit": {
```

Comment:
Namespace "flowManager.payloadSizeLimit" is probably more appropriate here - following our general pattern of "source.purpose" or "source.target" in naming